docs/firmware/zephyr-device-sdk/authentication/certificate-auth.md

```diff
@@ -186,7 +186,13 @@ With the public root certificate uploaded to Golioth, and the device
 certificate/key in use in your firmware, your device will automatically connect
 and authenticate to your project. Look in the [Golioth Web
 Console](https://console.golioth.io) to verify a device matching the
-`PRIMARY_HARDWARE_ID` you supplied when creating device credentials is now
+`DEVICE_NAME` you supplied when creating device credentials is now
```

Comment:
`DEVICE_NAME` is no longer used, so we are actually looking for a device name that has a prefix that matches the `CERTIFICATE_ID`.

Reply:
Yeah, that's true, but as we decide to remove the CertificateID field from the index table, we need to identify the recently provisioned device using some attribute. For this case, as we provision a new device with the same value for Device Name and CertificateID, we can use the device name. Because of that, i left explicit that the device is a mutable attribute and can be changed at any time. But for the first time the user can use it to find the new device.

Reply:
@miguel91it I don't think we need to add the certificate ID to the index table, but this sentence specifically says "matching the `DEVICE_NAME` you supplied" and the user is not supplying a `DEVICE_NAME` in this flow. I think just updating it to say something like "to verify a device with a name that matches or is prefixed by the `CERTIFICATE_ID` you supplied".

Reply:
Perfect, thanks Dan!